src/LinuxTracing/LinuxTracingIntegrationTestPuppet.cpp

```diff
@@ -60,6 +65,8 @@ int LinuxTracingIntegrationTestPuppetMain() {
     LOG("Puppet received command: %s", command);
     if (command == PuppetConstants::kSleepCommand) {
       SleepRepeatedly();
+    } else if (command == PuppetConstants::kSetnameCommand) {
+      ChangeName();
```

Comment:
ChangeCurrentThreadName?

Reply:
`ChangeMainThreadName`?

Reply:
Well ok, it can be called from anywhere, I'll change to `ChangeCurrentThreadName`.

Reply:
Done.